src/ui/colorui.js

```diff
@@ -117,7 +135,60 @@ export default class ColorUI extends Plugin {
 				editor.editing.view.focus();
 			} );
 
+			if ( documentColorsCount !== 0 ) {
+				dropdownView.on( 'change:isOpen', ( evt, name, val ) => {
+					if ( val ) {
+						const model = editor.model;
+						const document = model.document;
+						const rootNames = document.getRootNames();
```

Comment:
This variable should be inlined, it's used only once in for of loop. While extracting `maxCount` (that is in the following line and also occurs only once) sounds like a good idea (increases readability) this variable doesn't have this benefit.